plugins/se.infomaker.ximconcept/components/ConceptMainComponent.js

```diff
@@ -66,7 +66,8 @@ class ConceptMainComponent extends Component {
                 } else if (e.data.action === 'delete-all' && (associatedWith.length && associatedWith === eventName)) {
                     ConceptService.removeAllArticleLinksOfType(this.state.conceptType)
                 } else if (eventName === this.state.name || cleanEventName === this.state.name || matchingType) {
-                    this.reloadArticleConcepts()
+                    const updatedConcept = (e.data && e.data.data) ? e.data.data : false
```

Comment:
Default value for `updatedConcept` should probably match the default value used in `reloadArticleConcepts()`

Reply:
👍